themes/bootstrap3/templates/myresearch/holds.phtml

```diff
@@ -98,6 +98,35 @@
               <br />
             <? endif; ?>
 
+            <?
```

Comment:
What's all this about? There seem to be some related changes in multiple myresearch templates, and it's not clear what any of these have to do with the modern theme. Do you know what's going on here, or is this some kind of merge-related glitch?